cmd/check.go

```diff
@@ -70,8 +71,10 @@ Flags:
 			Packages:               checkflags.Packages,
 		}
 
-		// run all dynamically built audits in the auditor workqueue
-		capAuditor.RunAudits()
+		// run all dynamically built audits in the auditor workqueue.
+		if err := capAuditor.RunAudits(); err != nil {
```

Comment:
Rather than add a block just to add a debug log (especially a debug log that says "error"), it should probably just be a RunE and do a `return capAuditor.RunAudits()`.

Reply:
Not a blocker.